tests/drivers/uart/uart_async_api/boards/xg29_rb4412a.overlay

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2025, Silicon Laboratories Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*
+ * Need to connect PB0 and PB1 of the Expension Pin header
```

Comment:
Comment says PB0 and PB1, but pinctrl assigns PB2 and PB3.
These pins are also different from the GPIO loopback test above. It would be useful if the same loopback connection was used for as many tests as possible, and that different loopback pairs never conflict.